about/billing.html.markerb

```diff
@@ -0,0 +1,63 @@
+---
+title: Fly.io Billing
+layout: docs
+sitemap: true
+nav: firecracker
+redirect_from: /docs/about/credit-cards/
+---
+
+Billing for Fly.io is monthly per organization. Organizations are administrative entities that enable you to add members, share app development environments, and manage billing separately.
+
+If you provision services beyond the [free allowances](/docs/about/pricing/#free-allowances), we will charge you actual money at the end of each month.
+
+Refer to our [Pricing page](/docs/about/pricing/) for resource cost details.
+
+## View invoices
+
+You can view and download invoices from the Billing section of your [dashboard](https://fly.io/dashboard/personal/billing) for each organization.
+
+To download a past invoice, click View for the relevant cycle. You'll be sent to the Stripe billing portal where you'll have the option to download the invoice.
+
+## Volume billing
+
+Volume billing is pro-rated to the hour and we subtract the free allowances first. For details, see [Volume pricing](/docs/about/pricing/#persistent-storage-volumes).
+
+You aren't billed for the 5 days worth of [volume snapshots](/docs/apps/volume-manage/#restore-a-volume-from-a-snapshot) that we store for you.
```

Comment:
awesome, nitpick: we might want to make sure the wording here ensure that it's 1 per day so when we introduce on-demand snapshots it doesnt feel like all snapshots first 5 days are free but that can be added later